src/components/Tree/child.js

```diff
@@ -23,7 +24,7 @@ export default function Child(props) {
         onSelect,
     } = props;
     const hasChildren = Array.isArray(children);
-    const hasCheckbox = typeof isChecked === 'boolean';
+    const hasCheckbox = isChecked !== undefined;
```

Comment:
I think we should do:
```const hasCheckbox = typeof isChecked === 'boolean' || isChecked === 'indeterminate'```
in order to avoid render the checkbox when pass a wrong value that is different to `undefined`